packages/components/src/HStack/__tests__/__snapshots__/HStack.test.js.snap

```diff
@@ -90,15 +90,8 @@ exports[`props should render Spacer 1`] = `
   font-size: 13px;
   font-size: var(--wp-g2-font-size);
   margin: 0;
-  display: block;
-  max-height: 100%;
-  max-width: 100%;
-  min-height: 0;
-  min-width: 0;
-  display: var(--wp-g2-flex-item-display);
-  -webkit-flex: 1;
-  -ms-flex: 1;
-  flex: 1;
+  margin-bottom: calc(4px * 2);
```

Comment:
This was a bug fix 🙂 Previously `hasNamespace` did not work correctly 😱 This snapshot change is evidence that it is now working, where spacers are receiving only their styles and not the flex item styles.